matter_server/client/client.py

```diff
@@ -156,6 +157,49 @@ async def open_commissioning_window(
             ),
         )
 
+    async def get_matter_fabrics(
+        self, node_id: int
+    ) -> list[tuple[int, int, int, str | None, str | None]]:
+        """
+        Get Matter fabrics from a device.
+
+        Returns a list of tuples containing:
```

Comment:
maybe create a simple model (dataclass) for this ?

Reply:
Yeah it initially didn't return that much data ill fix that